.drone.yml

```diff
@@ -3,33 +3,263 @@ kind: pipeline
 type: docker
```

Comment:
why do we have a .yml and .star ?

Reply:
the repo seems to use the `.yml` file for ci, it ignores if only the `.star` is pushed. I've used the `.star` file for generating purposes only.

Reply:
that is easy to change, I've done that. the .star file should be used now